test/test_examples_2d_unstructured_quad.jl

```diff
@@ -49,6 +49,21 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
       linf = [0.31880214280781305, 0.3468488554333352, 0.34592958184413264, 0.784555926860546],
       tspan = (0.0, 1.0))
   end
+
+  @testset "elixir_advection_unstructured_quad_basic.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_unstructured_quad_basic.jl"),
+      l2   = [0.00023219572238346008],
+      linf = [0.0017401556568237275])
+  end
+
+  @testset "elixir_ape_unstructured_quad_gauss_wall.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_ape_unstructured_quad_gauss_wall.jl"),
+      l2   = [0.028478721740084537, 0.028283869218625128, 0.06602783645566432, 0.0,
+              7.465932985352019e-16, 1.4931865970704038e-15, 1.4931865970704038e-15],
+      linf = [0.264595205647784, 0.2618426663784852, 1.1513244355909877, 0.0,
+              8.881784197001252e-16, 1.7763568394002505e-15, 1.7763568394002505e-15],
+      tspan = (0.0, 10.0))
+  end
```

Comment:
How long does this test take (when running it for the first time)?

Reply:
From a fresh REPL this takes about 5 seconds. Should I shrink the `tspan`?

Reply:
Yeah, maybe a bit. How many time steps (RHS evaluations) does it take?

Reply:
It takes 46 time steps. That was the only metric I used to pick the final time. I forgot about the startup time from a fresh REPL

Reply:
I think a final time 5.0 would still be okay (the RK method has 9 stages).

Reply:
okay, I will update the test